### ISSUE TITLE
Solution Build fails for projects Dicom.Functions.App and Dicom.Web | [ Commit 511a865]

### ISSUE THREAD

**Issue by IoTFier** [microsoft/dicom-server#2365]
**Describe the bug**
Solution build fails with no containers deployed in docker.

**To Reproduce**
1.  Update Microsoft.Health.Dicom repo to the commit -
     [511a865 - Bump HealthcareSharedPackageVersion from 6.2.33 to 6.2.34 (#2356)]
2. Open the solution in VS Studio 17.4.5
3. Build solution

**Expected behavior**
Solution should build successfully with four containers running in Docker for Desktop

**Actual behavior**
A clear and concise description of what actually happened.

Build fails with error shown below.
I also noticed the Dockerfile under functions folder in docker-compose is missing. The file icon shows a dotted file with red cross.
![image](https://user-images.githubusercontent.com/26844611/219280552-103aeb2e-a026-4f69-8003-86cce4ddeb0d.png)

Build started...
1>------ Build started: Project: docker-compose, Configuration: Debug Any CPU ------
1>{path}\Microsoft\dicom-server\src\Microsoft.Health.Dicom.Functions.App\Microsoft.Health.Dicom.Functions.App.csproj : error MSB4057: The target "DockerResolveAppType" does not exist in the project.
1>Done building project "Microsoft.Health.Dicom.Functions.App.csproj" -- FAILED.
1>{Path}\Microsoft\dicom-server\src\Microsoft.Health.Dicom.Web\Microsoft.Health.Dicom.Web.csproj : error MSB4057: The target "DockerResolveAppType" does not exist in the project.
1>Done building project "Microsoft.Health.Dicom.Web.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 27 up-to-date, 0 skipped ==========
========== Elapsed 00:00.648 ==========

**Comment by wsugarman** [comment]
This is related to a recent change I made to better standardize the build property resolution. The use of `TargetFrameworks` in the function app and the web app seem to be confusing the docker compose project. Furthermore, I moved the function dockerfile after seeing that Azure Functions [are now supported for local debugging with containers](https://learn.microsoft.com/en-us/visualstudio/containers/edit-and-refresh?view=vs-2022). However, they are not seemingly supported by docker compose projects yet.

I have a pull request for reverting these two changes, and the docker-compose project should now work. Apologies for the breakage.
